### ISSUE TITLE
S16B: Passing Shot 4 player versions do not boot

### ISSUE THREAD

**Issue by jotego** [jotego/jtcores#159]
commit 78ee7cf

- passshta 
- cencourt
- passshtj

The System 16A version works
- passht16a

The System 16B version (2 players) works
- passsht

**Comment by jotego** [comment]
`pcb_5358L` in jts16b_main was not asserted for these games and it had to. Asserting it fixes the problem on passshta and passshtj. cencourt needs something else

```
always @(posedge clk) begin
    pcb_5358L <= game_id==8'h10 || game_id==8'h13 || game_id==8'h15 || game_id==8'h1a;
end
```
Partial fix in 88a5d8de

**Comment by jotego** [comment]
For cencourt the fix was changing the ID byte to the same as the rest of the series:

```
    { setname="cencourt",   offset=0x18, data="13" },
```

in the TOML.

Fixed in 00bea4b6
